### ISSUE TITLE
Latest version requires Java 14

### ISSUE THREAD

**Issue by smuldr** [utwyko/KMinRandom#6]
Our project fails to compile the tests after updating kMinRandom from 1.0.2 to 1.0.3. We use JDK 11 but it seems like kMinRandom now requires Java 14? 

Here's the Gradle build output:
```
> Task :my-project:kaptGenerateStubsTestKotlin FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':my-project:kaptGenerateStubsTestKotlin'.
> Could not resolve all files for configuration ':my-project:testCompileClasspath'.
   > Could not resolve nl.wykorijnsburger.kminrandom:kminrandom:1.0.3.
     Required by:
         project :my-project
      > No matching variant of nl.wykorijnsburger.kminrandom:kminrandom:1.0.3 was found. The consumer was configured to find an API of a library compatible with Java 11, preferably in the form of class files, and its dependencies declared externally, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'jvm' but:
          - Variant 'apiElements' capability nl.wykorijnsburger.kminrandom:kminrandom:1.0.3 declares an API of a library, packaged as a jar, and its dependencies declared externally, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'jvm':
              - Incompatible because this component declares a component compatible with Java 14 and the consumer needed a component compatible with Java 11
          - Variant 'runtimeElements' capability nl.wykorijnsburger.kminrandom:kminrandom:1.0.3 declares a runtime of a library, packaged as a jar, and its dependencies declared externally, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'jvm':
              - Incompatible because this component declares a component compatible with Java 14 and the consumer needed a component compatible with Java 11
```

**Comment by utwyko** [comment]
Thanks for filing the issue. I'm able to reproduce this and will investigate.

**Comment by utwyko** [comment]
This should be fixed in v1.0.4 which will be available shortly.
